SConstruct

```diff
@@ -478,8 +475,22 @@ for env in [test_env, client_env, env]:
         if env['sanitize']:
             env.AppendUnique(CCFLAGS = ["-fsanitize=" + env["sanitize"]], LINKFLAGS = ["-fsanitize=" + env["sanitize"]])
         
+# #
+# Start determining options for debug build
+# #
+        
         env["DEBUG_FLAGS"] = Split("-O0 -DDEBUG -ggdb3")
         
+        if env["glibcxx_debug"] == True:
+            env["GLIBCXX_DEBUG"] = Split("_GLIBCXX_DEBUG _GLIBCXX_DEBUG_PEDANTIC")
```

Comment:
Please just use a variable for this instead of messing with the environment. (Python has no `if` scope, a variable you define here is accessible anywhere in this file.)

    glibcxx_debug_flags = Split("_GLIBCXX_DEBUG GLIBCXX_DEBUG_PEDANTIC")